src/client/client.js

```
@@ -25,8 +25,17 @@ const initialState = {};
      * During the production client bundle build, the server will need to be
      * stopped.
      */
-    // eslint-disable-next-line no-underscore-dangle
+    /* eslint-disable no-underscore-dangle */
     StyleSheet.rehydrate(window.__APHRODITE__);
+    if (window.__ACTION__.sentry) {
```

remove the conditional, since we want it to fail loudly.